markdown/reference/sql/CREATE-CAST.html.md.erb

```diff
@@ -0,0 +1,101 @@
+---
+title: CREATE CAST
+---
+
+Defines a new cast.
+
+## <a id="topic1__section2"></a>Synopsis
+
+``` pre
+CREATE CAST (sourcetype AS targettype) 
```

Comment:
Global edit:  need to put < and > around variables, like <sourcetype> <targettype> <funcname> <argtypes> in keeping with the other reference pages.